README.md

```diff
@@ -30,11 +30,11 @@ pip install https://github.com/digiaiiris/zabbix-azure-monitoring/releases/downl
 
 Item Syntax | Description | Units |
 ----------- | ----------- | ----- |
-azure.insights.discover_insights[configuration_file, resource_group, resource] | Discover metrics in Application Insights resources | {#METRIC_CATEGORY}, {#METRIC_NAME} |
+azure.discover[configuration_file, resource_group, provider_name, resource_type, resource] | Discover metrics from Azure resources | {#METRIC_CATEGORY}, {#METRIC_NAME} |
```

Comment:
Wouldn't it be nice to have resource parameters in the output so the discovered items get their arguments from discovery?

Also this discovers metrics for specific resource. Should there be discovery of resources of particular type?